libr/core/cmd_write.c

```diff
@@ -315,8 +315,8 @@ static void cmd_write_value (RCore *core, const char *input) {
 	if (input && input[0] && input[1] && input[2]) {
 		off = r_num_math (core->num, input+2);
 	}
-	if (core->file) {
-		r_io_use_desc (core->io, core->file->desc);
+	if (core->file && core->file->desc) {
```

Comment:
maybe is better to not change the signature of r_io_use_desc and add r_io_use_desc_fd